models/articles-models.js

```diff
@@ -40,4 +40,24 @@ exports.selectArticleComments = (param_id) => {
     }).then(({rows})=> {
         return rows
     })
+}
+
+exports.insertArticleComment = (articleId, {username, body}) => {
+    if (!username || !body) return Promise.reject({status: 400, msg: "bad request"})
```

Comment:
This would be handled by PSQL if the columns are not null! You could see if PSQL throws an error that you can catch instead of doing this.